desktop/core/src/desktop/lib/raz/raz_client.py

```diff
@@ -38,7 +38,7 @@
   from urllib import unquote as lib_urlunquote
 
 
-LOG = logging.getLogger(__name__)
+LOG = logging.getLogger()
```

Comment:
@ranade1 I believe this needs to be done for every file wherever `LOG = logging.getLogger(__name__)` is present else those specific logs will not be registered.

Reply:
Going forward, we will patch it as needed.